manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ErrorListResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.redhat.service.smartevents.manager.api.models.responses;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import com.redhat.service.smartevents.infra.api.models.responses.PagedListResponse;
+
+@Schema
+public class ErrorListResponse extends PagedListResponse<ErrorResponse> {
```

Comment:
We already have a `com.redhat.service.smartevents.infra.api.models.responses.ErrorListResponse`

Reply:
Changed all these names to `ProcessingError*`. Thanks for pointing this out :+1: